manifests/charts/istiod-remote/values.yaml

```diff
@@ -108,5 +108,7 @@ pilot:
   configMap: true
 global:
 
+  # One external istiod controls all remote clusters: disabled by default
```

Comment:
Can we remove one here?  It could be multiple istiod replicas.

Reply:
removed